src/main/java/mate/academy/spring/service/impl/MovieSessionMapperImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.spring.service.impl;
+
+import mate.academy.spring.dto.MovieSessionRequestDto;
+import mate.academy.spring.dto.MovieSessionResponseDto;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionMapperImpl {
```

Comment:
```suggestion
public class MovieSessionMapper {
```
same for other mappers